src/System.ComponentModel.TypeConverter/tests/Drawing/ImageConverterTests.cs

```diff
@@ -0,0 +1,354 @@
+        public void ConvertFrom_SizeF()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConv.ConvertFrom(null, CultureInfo.InvariantCulture, new SizeF(10, 10)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void ConvertFrom_Object()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConv.ConvertFrom(null, CultureInfo.InvariantCulture, new object()));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertFrom_BadString()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertFrom("System.Drawing.String"));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertFrom_BadString_Culture()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, "System.Drawing.String"));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertFrom_Bitmap()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, new Bitmap(20, 20)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertFrom_Point()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, new Point(10, 10)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertFrom_SizeF()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, new SizeF(10, 10)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TypeDescriptor_ConvertFrom_Object()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, new object()));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void ConvertTo()
+        {
+            Assert.Equal(_imageStr, (string)_imgConv.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(string)));
+            Assert.Equal(_imageStr, (string)_imgConv.ConvertTo(_image, typeof(string)));
+            Assert.Equal(_imageStr, (string)_imgConvFrmTD.ConvertTo(null, CultureInfo.InvariantCulture, _image, typeof(string)));
+            Assert.Equal(_imageStr, (string)_imgConvFrmTD.ConvertTo(_image, typeof(string)));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void ConvertTo_ByteArray()
+        {
+            byte[] newImageBytes = (byte[])_imgConv.ConvertTo(null, CultureInfo.InvariantCulture,
+                _image, _imageBytes.GetType());
+
+            Assert.Equal(_imageBytes.Length, newImageBytes.Length);
```

Comment:
Should we also assert for the content inside the new byte[] vs the old? to make sure we're getting the right bytes?